gluten-ut/spark32/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala

```diff
@@ -54,7 +54,9 @@ class VeloxTestSettings extends BackendTestSettings {
       "SPARK-32038: NormalizeFloatingNumbers should work on distinct aggregate",
       // Replaced with another test.
       "SPARK-19471: AggregationIterator does not initialize the generated result projection" +
-        " before using it"
+        " before using it",
+      // TODO: fix inconsistent behavior.
+      "SPARK-17641: collect functions should not collect null values"
```

Comment:
Could you paste the relevant issue here?

Reply:
https://github.com/oap-project/gluten/issues/4763